BUILD

```diff
@@ -21,25 +23,29 @@ alias(
     actual = "//tools/format:format.fix",
 )
 
-alias(
+checker(
     name = "copyright.check",
```

Comment:
.check and .fix will always be instantiated with the same input, best to hide those under the one macro.

Reply:
I'm not sure if I understood what you want. Should the macro expand both targets? Check the update; if this is not what you want, let me know.

Reply:
Yes, exactly, thanks.